src/Sulu/Bundle/AdminBundle/Resources/js/views/List/List.js

```diff
@@ -1,129 +1,53 @@
 // @flow
-import {observable} from 'mobx';
 import {observer} from 'mobx-react';
 import React from 'react';
+import Datagrid from '../../containers/Datagrid';
+import DatagridStore from '../../containers/Datagrid/stores/DatagridStore';
+import resourceMetadataStore from '../../stores/ResourceMetadataStore';
 import {translate} from '../../services/Translator';
 import {withToolbar} from '../../containers/Toolbar';
-import {Table, Body, Header, Cell, HeaderCell, Row} from '../../components/Table';
 import type {ViewProps} from '../../containers/ViewRenderer/types';
 
 @observer
 class List extends React.PureComponent<ViewProps> {
-    page = observable();
-    @observable tableData = {
-        header: [
-            'Type of',
-            'Name',
-            'Author',
-            'Date',
-            'Subversion',
-            'Uploadgröße',
-        ],
-        body: [
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-            [
-                'Blog',
-                'My first 100 day in Vorarlberg',
-                'Adrian Sieber',
-                '24.12.2017',
-                'Github',
-                '20 MB',
-            ],
-        ],
-    };
+    datagridStore: DatagridStore;
 
     componentWillMount() {
-        this.props.router.bindQuery('page', this.page, '1');
+        const router = this.props.router;
+        const {
+            route: {
+                options: {
+                    resourceKey,
```

Comment:
what if `resourceKey` doesn't exist in the route options? error handling?

Reply:
Good catch, I think I am throwing an error in that case. But I think that should also be handled somehow in our testing infrastructure... We need something similar as the behat tests now (but please no behat...), which just try to call all lists, and see if they are not failing.